end_to_end/0-AutoClaimFraudDetection.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<p>remove "overview"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>no. we need to be able to navigate back to the top at any time from any section :-)</p>